added a root admin with name admin and empty password

Ica-scorp opened the pull request.

HaotingShen commented:
After logging in as admin, only two options are displayed. There should be four.
Also, the private options are missing two loan options.

Running the program...
1. Login to account    2. Create account
3. Reset Password      4. Exit
Enter a number [1-4]:
1
Enter username:
admin
Enter password:

Login successful!
1. Print All Transaction    2. Recall Transaction
Enter a number [1-2]:

Ica-scorp commented:
Sorry for that. I forgot to account for the newly changed options and just added them as well. I deleted one of the disable admin panel option for admin so we can somehow separate admins from normal users (and they do not have transactions). If that does not makes sense I will revert this change back!

HaotingShen commented:
> Sorry for that. I forgot to account for the newly changed options and just added them as well. I deleted one of the disable admin panel option for admin so we can somehow separate admins from normal users (and they do not have transactions). If that does not makes sense I will revert this change back!

I personally think treating the admin as a special user is a good idea, and it makes the design more straightforward and easier for peers to use.
One thing to note: the admin menu currently doesn't have a Logout option, so the admin can't log out right now lol.

Ica-scorp commented:
Good catch! Justed added the logout option here.

HaotingShen reviewed (approved):
Looks good to me!
